presto-product-tests/src/main/java/io/prestosql/tests/jdbc/TestLdapPrestoJdbc.java

```diff
@@ -81,7 +82,7 @@ public void shouldFailQueryForWrongLdapPassword()
     @Test(groups = {LDAP, PRESTO_JDBC, PROFILE_SPECIFIC_TESTS}, timeOut = TIMEOUT)
     public void shouldFailQueryForWrongLdapUser()
     {
-        expectQueryToFail("invalid_user", ldapUserPassword, "Authentication failed: Access Denied: Invalid credentials");
+        assertTrue(getQueryFailureMessage("invalid_user", ldapUserPassword).startsWith("Authentication failed"));
```

Comment:
Should the message not match the expected condition, the assertion will propagate useless test message's failure.

Use `org.assertj.core.api.Assertions#assertThatThrownBy(...)` instead.


Then you can probablt revert change in `presto-product-tests/src/main/java/io/prestosql/tests/jdbc/BaseLdapJdbcTest.java`